concrete/src/Cache/Adapter/ZendCacheDriver.php

```diff
@@ -87,12 +138,14 @@ protected function internalGetItem(&$normalizedKey, &$success = null, &$casToken
      */
     protected function internalSetItem(&$normalizedKey, &$value)
```

Comment:
This is part of zend yeah? I wonder why they pass both arguments by reference :S